secretsmanager/src/misc.h

```diff
@@ -0,0 +1,69 @@
+// This implementation is copied from CryptoPP: https://www.cryptopp.com/wiki/OPENSSL_EVP_BytesToKey
```

Comment:
consider moving to utils

Reply:
This algorithm is deprecated. It will be removed in future PR.